src/wled/models.py

```diff
@@ -727,13 +727,17 @@ class Device(BaseModel):
     @classmethod
     def __pre_deserialize__(cls, d: dict[Any, Any]) -> dict[Any, Any]:
         """Pre deserialize hook for Device object."""
-        if (version := d.get("info", {}).get("ver")) and version < MIN_REQUIRED_VERSION:
-            msg = (
-                f"Unsupported firmware version {version}. "
-                f"Minimum required version is {MIN_REQUIRED_VERSION}. "
-                f"Please update your WLED device."
-            )
-            raise WLEDUnsupportedVersionError(msg)
+        version = d.get("info", {}).get("ver")
+        if version:
+            # Strip any trailing characters after a dash (e.g., "-b1")
+            clean_version = version.split('-')[0]
```

Comment:
This is not correct. Beta versions **are** older versions.

Reply:
Yes, but:
- The trailing characters **break** the comparison, not just return the wrong result
- A beta version should see the API for the proposed version, even though it is older than the eventual release 
- you are broken "out-of-the-box" with the athom (https://www.athom.tech/blank-1/wled-high-power-led-strip-controller) as shipped

Reply:
> A beta version should see the API for the proposed version, even though it is older than the eventual release

Unfortunately, that is not the case.

> you are broken "out-of-the-box" with the athom ([athom.tech/blank-1/wled-high-power-led-strip-controller](https://www.athom.tech/blank-1/wled-high-power-led-strip-controller)) as shipped

Honestly, you should contact Athom about that. Shipping beta firmware IMHO as a final product seems wrong. Additionally, 0.14.0 is REALLY OLD...

../Frenck